backend/bamboo/models.py

```diff
@@ -0,0 +1,269 @@
+from datetime import datetime
+
+from sqlalchemy import Integer, String, Text, ForeignKey, Boolean, DateTime, Float
+from sqlalchemy.orm import relationship
+from werkzeug.security import generate_password_hash, check_password_hash
+from sqlalchemy.orm import Mapped, mapped_column
+
+from bamboo.core.extensions import db
+
+
+class User(db.Model):
+    id: Mapped[int] = mapped_column(Integer, primary_key=True)
```

Comment:
SQLAlchemy 2.x automatically set the data type with type hint, so the declaration can be simplified to `id: Mapped[int] = mapped_column(primary_key=True) name: Mapped[str] username: Mapped[str] ...`

Reply:
Updated.